pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -41,8 +41,7 @@ type EventDispatcher interface {
 
 	// PostStopPod is invoked after all of a pod's containers have permanently
 	// stopped running, but before the pod sandbox is destroyed.
-	PostStopPod(pod *v1.Pod, cgroupPath string) (*lifecycle.EventReply, error)
-
+	PostStopPod(cgroupPath string) error
```

Comment:
Basically you need to update this functions: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_pods.go#L1549 and https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet_pods.go#L802 to pass information about destroyed pods. L802 could be tricky one to gather this information from corner cases.
Moreover you need to extend one(or two) interfaces.
For me it's not so easy. @PiotrProkop any thoughts of wisdom?